lib/compress.js

```diff
@@ -4174,8 +4174,25 @@ merge(Compressor.prototype, {
         }
     }
 
+    function use_increment(self) {
+        if (self.step instanceof AST_Assign
+            && self.step.operator === "+="
+            && self.step.left instanceof AST_Symbol
+            && self.step.right.print_to_string() === "1"
```

Comment:
`print_to_string` is very inefficient. Please check the specific AST node type - AST_Constant with value === 1.